src/Mvc/Extensions.ApiDescription.Client/src/Microsoft.Extensions.ApiDescription.Client.nuspec

```diff
@@ -21,9 +21,5 @@
     <file src="buildMultiTargeting\*" target="buildMultiTargeting" />
     <file src="bin\$configuration$\net461\Microsoft.Extensions.ApiDescription.Tasks.*" target="tasks\net461" />
     <file src="bin\$configuration$\netstandard2.0\Microsoft.Extensions.ApiDescription.Tasks.*" target="tasks\netstandard2.0" />
-    <file src="..\..\dotnet-getdocument\src\bin\$configuration$\netcoreapp2.1\publish\*.*" target="tools" />
```

Comment:
What happened to these?

Reply:
They aren't necessary with the new client / server split. The tool will come back in Microsoft.Extensions.ApiDescription.Server